components/header/userMenu/index.tsx

```diff
@@ -37,55 +38,75 @@ export default function UserMenu(): ReactElement {
   const toggleMenu = () => setUserMenuOpen(!userMenuOpen);
 
   return (
-    <div
-      className={bem("header-banner__aside-menu", "popup")}
-      onMouseEnter={toggleMenu}
-      onMouseLeave={toggleMenu}
-    >
-      <button
-        className={bem("header-banner__aside-menu-trigger")}
-        type="button"
-        aria-haspopup="true"
-        aria-controls="UserMenu"
-        aria-expanded={!userMenuOpen}
-        onClick={toggleMenu}
-      >
-        <i
-          className={clsx(
-            bem("icon-user"),
-            bem("header-banner__aside-menu-trigger-icon")
-          )}
-          aria-label="User Menu"
-        />
-      </button>
+    <>
+      <div className={bem("header-banner__main-nav")}>
+        <Link href="/contacts" replace>
```

Comment:
I think this needs to be replaced with the Main Menu pattern in Prism - but we can do it in another PR, as we probably want to use the Main Menu as a React component instead.